CODEOWNERS

```diff
@@ -31,6 +31,7 @@
 /connectivity/check/ipcache.go @cilium/ipcache
 /connectivity/check/metrics*.go @cilium/metrics
 /connectivity/check/policy.go @cilium/sig-policy
+/connectivity/factory/factory.go @cilium/ci-structure
```

Comment:
I think @tklauser wanted to be more specific about the relevant tests. Like `/connectivity/factory/echo_ingress_knp.go`
I guess we can derive the names of the tests from git blame in the main branch.

Reply:
Ah, my bad! Fixed.

Reply:
Yep, that's what I had in mind. Thanks for pointing it out @nickolaev and thanks for the fix @viktor-kurchenko.

Reply:
> I guess we can derive the names of the tests from git blame in the main branch.

Agree, I think we should already add the respective teams as owners where appropriate. Also, I think we don't need an entry for each file, we can just add `/connectivity/factory/* @cilium/ci-structure` as a catch-all for all files that are not covered by more specific entries.

Reply:
Both :) the catch all for any file that doesn't show a clear owner in `git blame`.

Reply:
I think it's a good start though.